library/CMakeLists.txt

```diff
@@ -249,6 +250,9 @@ ELSE(WIN32)
   SET(PROJECT_LIBS psapi dfhack-tinyxml dfhack-tinythread)
 ENDIF()
 
+ADD_LIBRARY(dfhack-version STATIC DFHackVersion.cpp)
+SET_TARGET_PROPERTIES(dfhack-version PROPERTIES COMPILE_DEFINITIONS DFHACK_VERSION="${DFHACK_VERSION}")
+
```

Comment:
This should work on Windows, but it would be nice if somebody could test it before merging.